modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -1354,29 +1356,33 @@ def stack(self, level, dropna):
         else:
             new_columns = None
 
-        new_modin_frame = self._modin_frame.apply_full_axis(
-            1,
+        new_modin_frame = self._modin_frame.map(
             lambda df: pandas.DataFrame(df.stack(level=level, dropna=dropna)),
+            axis=1,
+            full_axis=True,
             new_columns=new_columns,
         )
         return self.__constructor__(new_modin_frame)
 
     # Map partitions operations
     # These operations are operations that apply a function to every partition.
-    abs = Map.register(pandas.DataFrame.abs, dtypes="copy")
+    # Though all these operations are element-wise, some can be sped up by mapping across a row/col
```

Comment:
My quick and dirty micro-benchmarks show no difference between specifying an axis vs. applying cell-wise, so perhaps it's best to revert back to cell-wise operations. The hope is that for certain operators, being able to apply across a whole axis rather than having to examine each cell would provide a speedup. I will see if any other benchmarks would justify this theory.